certified-connectors/PartnerCenterReferrals/apiDefinition.swagger.json

```diff
@@ -243,6 +243,30 @@
                   "items": {},
                   "title": "Offers",
                   "type": "array"
+                },
+                "additionalRequirements": {
+                  "description": "Additional Requirements",
+                  "items": {
+                    "properties": {
+                      "attributes": {
+                        "description": "Attributes",
+                        "title": "Attributes",
+                        "properties": {
+                          "id": {
+                            "description": "Id",
+                            "title": "Id",
+                            "type": "string"
+                          },
+                          "type": {
+                            "description": "Type",
+                            "title": "Type",
+                            "type": "string"
+                          }
+                        },
+                        "type": "array"
```

Comment:
Hello @SukruthMN, 

Can you please add items to the type array?

Reply:
Hi @vmanoharas ,
The array is closing statement for additional requirements item which as 2 components 'id' and 'type'. Do I need to do any changes to code?

Reply:
Hi @vmanoharas ,
As per above schema example, I have validated to submit changes should I need to raise new PR again, to submit changes , please advice? 

I need to update to:  "type": "object" instead of array.

Reply:
Hi @vmanoharas ,
Could you share update how to proceed further?
Thanks,

Reply:
Hi @SukruthMN, 

My apology for not responding.  Please close this PR and re-submit new PR.